app/workers/migrate_application_choices_worker.rb

```diff
@@ -0,0 +1,37 @@
+class MigrateApplicationChoicesWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :low_priority
+
+  def perform(choice_ids)
+    errors = []
+
+    ApplicationChoice.where(id: choice_ids).find_each(batch_size: 100) do |choice|
+      application_form = choice.application_form
+
+      if choice.work_experiences.blank? && application_form.application_work_experiences.any?
+        choice.work_experiences = application_form.application_work_experiences.map(&:dup)
+      end
+
+      if choice.volunteering_experiences.blank? && application_form.application_volunteering_experiences.any?
+        choice.volunteering_experiences = application_form.application_volunteering_experiences.map(&:dup)
+      end
+
+      if choice.work_history_breaks.blank? && application_form.application_work_history_breaks.any?
+        choice.work_history_breaks = application_form.application_work_history_breaks.map(&:dup)
+      end
```

Comment:
Should we wrap this in a transaction?
If there is an error we wouldn't want any of these changes to persist